README.md

```diff
@@ -301,10 +301,10 @@ mod color;
 And finally the last feature! Just run the following command (note the `-m doc` at the end):
 
 ```shell
-cargo run --release -- -c YourGirFile.toml -d ../gir-files -o the-output-directory -m doc
+cargo run --release -- -c YourGirFile.toml -d ../gir-files --doc-target-path the-output-file-name -m doc
```

Comment:
The `-o` option isn't used for docs generation?

Reply:
It can be used, but file name will be "vendor.md"

Reply:
yes, that made things a bit awkward in gstreamer